reference/command/credentials.md

```diff
@@ -132,10 +132,8 @@ HELLO
 » ockam reset -y
 » ockam enroll
 
-» ockam project information --output json > project.json
-
-» ockam node create a --project-path project.json
-» ockam node create b --project-path project.json
```

Comment:
The --project-path argument doesn't exist anymore and we now have the ockam project import command. But why do we want to add this extra step if this example is not a multi-machine example?

Reply:
Yes that seems unnecessary in that case.